images/bigquery/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN apt-get update && apt-get install -y \
 
 RUN pip3 install influxdb==5.2.2 google-cloud-bigquery==0.24.0 ruamel.yaml==0.16
 
-ENV GCLOUD_VERSION 195.0.0
+ENV GCLOUD_VERSION 233.0.0
```

Comment:
Let's bump to latest. I verified this works *locally* for `metrics-bigquery` and `metrics-kettle` jobs.
```suggestion
ENV GCLOUD_VERSION 265.0.0
```

Reply:
done